cfpb/consumerfinance.gov
# Remove ArticlePage and ArticleLink

Found some unused CSS and realized this whole page type appeared unused and deprecated.

<img width="1241" alt="Screen Shot 2023-03-16 at 7 18 10 PM" src="https://user-images.githubusercontent.com/704760/225774140-3d56fd9b-9a4f-40a6-83d6-04738ce3906d.png">

## Removals

- Remove ArticlePage and ArticleLink and associated assets.

## How to test this PR

1. Run migrations and log into Wagtail and see that you can't create an Article page type any longer. 

Did I miss anything here??
